src/main/java/build/buildfarm/instance/shard/RedisShardBackplane.java

```diff
@@ -505,18 +505,34 @@ private void startFailsafeOperationThread() {
 
   @Override
   public void start(String clientPublicName) throws IOException {
-
     // Construct a single redis client to be used throughout the entire backplane.
     // We wish to avoid various synchronous and error handling issues that could occur when using
     // multiple clients.
     client = new RedisClient(jedisClusterFactory.get());
 
+    this.prequeue = createPrequeue(config);
+    this.operationQueue = createOperationQueue(config);
+
+    if (config.getSubscribeToBackplane()) {
+      startSubscriptionThread();
+    }
+    if (config.getRunFailsafeOperation()) {
+      startFailsafeOperationThread();
+    }
+
+    // Record client start time
+    client.call(
+        jedis -> jedis.set("startTime/" + clientPublicName, Long.toString(new Date().getTime())));
+  }
+
+  BalancedRedisQueue createPrequeue(RedisShardBackplaneConfig config) throws IOException {
```

Comment:
This looks like it qualifies for static

Reply:
changed the "hash generation" func to take the client as an argument, so these "queue creation" functions can also take the client and now be marked static.  
that's better.  when things are marked static they become obvious candidates to move out of the backplane in the future.